### ISSUE TITLE
When running existing project for the first time, developers don't know they should run `wasp db migrate-dev` and error message from Prisma about it is confusing.

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#248]
If a developer checks out a new wasp project and tries to run it for the first time with `wasp start`, they might get an error from prisma about needing to run `prisma generate` first.

This can be solved by running `wasp db migrate-dev` first, which we do mention in the documentation, however developers most often don't see that or forget it (which is completely reasonable, I also forget it and it is easy to oversee) so they get an error message about prisma about `prisma generate` which is rather cryptic in the context of wasp, especially since they can't really run ` prisma generate`.

Some solutions for this:
1. Somehow detect when this situation is happening and inform developer they need to run `wasp db migrate-dev`. Tricky part here might be detecting this situation, although I am pretty sure we can do it if we put enough effort into it.
2. Detect the situation and do it automatically for them? I am not sure if this can be done in reliable way at all, I actually think it probably can't, but it is worth giving it a big of consideration before giving up on it.
3. Maybe there is a different way to go about it -> maybe it would be better to just run `prisma generate` instead of `prisma migrate-dev`? Maybe we could do that automatically? So the idea here is to consider: what happens if we just run `prisma generate` instead of `prisma migrate-dev`, would that be easier / open up different ways to go about this?

**Comment by Martinsos** [comment]
@shayneczyzewski I just found this feature and I believe it is duplicate of what you are solving right now, so assigning it to you to figure out if you should also close it once your prisma PR is done!

**Comment by shayneczyzewski** [comment]
Sounds good @Martinsos This does appear to be a duplicate of the other two, so I will confirm once the other is done this is fully addressed and close if so.

**Comment by shayneczyzewski** [comment]
This should be addressed with a new warning we added when we detected the user may need to `db migrate-dev` on first run (or with updates after a migrate) [https://github.com/wasp-lang/wasp/pull/424]. Additionally, we now also automatically generate the Prisma client as part of overall app generation [https://github.com/wasp-lang/wasp/pull/444].

**Comment by Martinsos** [comment]
Awesome :)!!
